### PR TITLE
Update info on EKF tuning and baro usage

### PR DESCRIPTION
**Pull request by hamishwillee**
This falls out of discord discussion on how to disable baros and also how they are selected: https://discord.com/channels/1022170275984457759/1044214748050370630

### CODE REVIEW

File: en/advanced_config/tuning_the_ecl_ekf.md [hunk 1]
```diff
@@ -170,9 +170,13 @@ This reference is defined by the [EKF2_HGT_REF](../advanced_config/parameter_ref
 
 ### Barometer
 
-Enable/disable using [EKF2_BARO_CTRL](../advanced_config/parameter_reference.md#EKF2_BARO_CTRL).
+Enable/disable using [EKF2_BARO_CTRL](../advanced_config/parameter_reference.md#EKF2_BARO_CTRL) as a source for [Height](#height) data.
 
-For more details about the configuration of height sources, [click here](#height).
+Note that data from only one barometer is fused, even if multiple barometers are available.
+The barometer with the highest [CAL_BAROx_PRIO](../advanced_config/parameter_reference.md#CAL_BARO0_PRIO) priority value is selected first, falling back to the next highest priority barometer if a sensor fault is detected.
```
Comment (hamishwillee) [inline en/advanced_config/tuning_the_ecl_ekf.md:176]: @bresch in discord it said first instance is detected, but given we have priorities I assume that we use them in the selection? If we do, how do we select between equal priorities?

Comment (bresch) [reply en/advanced_config/tuning_the_ecl_ekf.md:176]: > given we have priorities I assume that we use them in the selection?

yes

> If we do, how do we select between equal priorities?

Then the first detected instance is used. This is kinda random so it's better to set the priorities properly.

**Review by bresch — APPROVED** [review]

File: en/advanced_config/tuning_the_ecl_ekf.md (hunk 1 above)
Comment (hamishwillee) [inline en/advanced_config/tuning_the_ecl_ekf.md:176]:
```suggestion
The barometer with the highest [CAL_BAROx_PRIO](../advanced_config/parameter_reference.md#CAL_BARO0_PRIO) priority value is selected first, falling back to the next highest priority barometer if a sensor fault is detected.
If barometers have equal-highest priorities, the first detected is used.
```

Comment (hamishwillee) [reply en/advanced_config/tuning_the_ecl_ekf.md:176]: Thanks! I'll add  a note.

**Pull request merged**